### DISCUSSION TITLE
Ability to flag spam + GH-wide monitoring spam accounts

### DISCUSSION THREAD

**Discussion by olets** [community/community#3781] [Discussions]
The introduction of Discussions seems to have brought in fake accounts and spam comments. It would be valuable to let all users flag such comments for moderator attention.

Beyond UI, it would be good track of the users and/or IPs leaving spam comments. A spam account could target unrelated repos, so a GitHub-wide solution is necessary to catch abuse. I'm not familiar with how to do that ethically. Platforms like Wikipedia and Stack Exchange would be good references.

---

Examples:

In both cases the initial comment did not include a link. An edit was made to add the link. Almost looks like someone testing a new bot's features.

- User https://github.com/Jaynesty

    https://github.com/tailwindlabs/tailwindcss/discussions/3878#discussioncomment-561843

    <kbd><img src="https://user-images.githubusercontent.com/3282350/114755370-8e342700-9d27-11eb-8cc6-771f32dcc680.png"></kbd>

- User https://github.com/Christopher784

    https://github.com/tailwindlabs/tailwindcss/discussions/3881#discussioncomment-558229

    <kbd><img src="https://user-images.githubusercontent.com/3282350/114755439-a015ca00-9d27-11eb-9814-b202e9636b47.png"></kbd>

**Comment by evi-liu** [comment]
Have you tried reporting the comments as "Spam"? We do our best to catch and hide spammy comments, however sometimes they'll sip through the cracks. If you ever see a spammy comment, you can report it as spam, which will hide the spammy content and flag the user in our system.

![image](https://user-images.githubusercontent.com/8809849/115933363-99800480-a443-11eb-82ae-395a369887b8.png)

**Reply by olets** [reply]
Thanks for the reply but I'm not sure this is answered, so I removed the "answer" mark you checked.

This option is not available to me on either of the examples I provided. Is that because I'm not a collaborator in the repo?

**Reply by TonyGravagno** [reply]
Not seeing Report Content either. Agreed that any medium that's exposed to the masses should have mechanisms that support automated and collaborative weeding out of bad users and content.

**Reply by dacap** [reply]
@evi-liu it looks like the option is not available anymore, and there are forums (like the ["Feed Feedback"](https://github.com/github/feedback/discussions/categories/feed-feedback)) completely full of spam.
